tck/src/main/java/com/akkaserverless/javasdk/tck/model/eventsourcedentity/EventSourcedTckModelEntity.java

```diff
@@ -43,12 +42,10 @@ public EventSourcedTckModelEntity(Context context) {
         context.serviceCallFactory().lookup(EventSourcedTwo.name, "Call", Request.class);
```

Comment:
TCK tests are failing here:
```
[info] 2021-08-24 14:28:11,451 ERROR com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntitiesImpl - Terminating entity due to unexpected failure
[info] java.lang.IllegalStateException: Service akkaserverless.tck.model.eventsourcedentity.EventSourcedTwo does not provide resolved methods and so can't be looked up by this factory
[info] 	at com.akkaserverless.javasdk.impl.ResolvedServiceCallFactory.lookup(ResolvedServiceCallFactory.scala:39)
[info] 	at com.akkaserverless.javasdk.tck.model.eventsourcedentity.EventSourcedTckModelEntity.<init>(EventSourcedTckModelEntity.java:42)
[info] 	at com.akkaserverless.javasdk.tck.model.eventsourcedentity.EventSourcedTckModelEntityProvider.newHandler(EventSourcedTckModelEntityProvider.java:71)
[info] 	at com.akkaserverless.javasdk.tck.model.eventsourcedentity.EventSourcedTckModelEntityProvider.newHandler(EventSourcedTckModelEntityProvider.java:30)
[info] 	at com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntitiesImpl.runEntity(EventSourcedEntitiesImpl.scala:145)
[info] 	at com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntitiesImpl.$anonfun$handle$1(EventSourcedEntitiesImpl.scala:125)
```

@pvlugter Do you have a clue of what is missing in ResolvedServiceCallFactory when using the new register method?

Reply:
I found and fixed that: https://github.com/lightbend/akkaserverless-java-sdk/pull/233/commits/007bc9094d217fe532cf86b062c75aecb6124581